sass/cd/cd-header/_cd-user-menu.scss

```diff
@@ -60,18 +49,8 @@
   color: var(--cd-white);
   background: transparent;
   line-height: var(--cd-global-header-height);
-
-  // Separator.
-  // &::after {
-  //   position: absolute;
-  //   top: 50%;
-  //   right: -1px;
-  //   width: 2px;
-  //   height: calc(var(--cd-global-header-height) / 3);
-  //   content: "";
-  //   transform: translateY(-50%);
-  //   background: var(--brand-grey);
-  // }
+  display: flex;
```

Comment:
it seems like `display: block` is declared before this one, so I moved these attributes higher in the list in my rebased PR.